CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Contributing
+
+CiviHR is an open-source project and we welcome new contributions from the community.
+
+## Label Types
+We use the following label types for issues and pull requests.
+
+#### Bug
```

Comment:
I think it makes more sense to have the labels as a list:
- **Bug**: blah blah blah...
- **Enhancement**: yada yada...
- **New Feature**: lore ipsum...